.github/workflows/dasboard-format.yaml

```
@@ -0,0 +1,43 @@
+name: KEEP Token Dashboard Code Format Checks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+defaults:
+  run:
+    working-directory: ./solidity/dashboard
+
+jobs:
+  dashboard-detect-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      dashboard: ${{ steps.filter.outputs.dashboard }}
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        filters: |
+          dashboard:
+            - './solidity/dashboard/**'
+
+  dashboard-code-format:
+    needs: dashboard-detect-changes
+    if: ${{ needs.dashboard-detect-changes.outputs.dashboard == 'true' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
```

The `setup-node` action now [supports caching](https://github.com/actions/setup-node#caching-packages-dependencies), we can add following:

```suggestion
          node-version: "12.x"
          cache: 'npm'
```